scripts/monthly_data_exports.js

```diff
@@ -71,7 +71,7 @@ const queries = [
     query: `
     SELECT
     t."createdAt", c.slug as "collective slug", t.type as "transaction type", t.amount::float / 100,
-    t.currency, fc.slug as "from slug", fc.type as "from type", t.description, e.category as "expense category",
+    t.currency, fc.slug as "from slug", fc.type as "from type", t.description, e.tags as "expense tags",
```

Comment:
Detail: the category was migrated as the first tag of the expense.
If you want to fetch that you can use `e.tags[1] as category`, otherwise the output will have the array of all tags displayed like `"[""OTHER""]"`.